Tools/ftscalingbench/ftscalingbench.py

```diff
@@ -37,11 +38,25 @@
 in_queues = []
 out_queues = []
 
-
 def register_benchmark(func):
     ALL_BENCHMARKS[func.__name__] = func
     return func
 
+
+@register_benchmark
+def shallow_copy():
+    x = [1, 2, 3]
+    shallow_copy = copy.copy
+    for i in range(200 * WORK_SCALE):
+        shallow_copy(x)
```

Comment:
Let's use more idiomatic Python, unless we need to specifically test something about this pattern:

```suggestion
    for i in range(200 * WORK_SCALE):
        copy.copy(x)
```